pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -10175,6 +10171,29 @@ def test_add_phase_center_known_source(sma_mir):
     assert return_id == 1
 
 
+def test_unphased_deprecation(sma_mir):
+    with uvtest.check_warnings(
+        DeprecationWarning,
+        "The `unphased` catalog type has been renamed to `unprojected`. Using "
```

Comment:
Might it be worth putting unphased/unprojected in these deprecation warnings here be in quotes instead of tics just so that it's clear it's a string and not an attribute of some variety?